kotlin/src/test/kotlin/io/smallrye/mutiny/coroutines/DeferredAsUniTest.kt

```diff
@@ -1,16 +1,14 @@
 package io.smallrye.mutiny.coroutines
 
 import io.smallrye.mutiny.helpers.test.UniAssertSubscriber
-import kotlinx.coroutines.CancellationException
-import kotlinx.coroutines.ExperimentalCoroutinesApi
-import kotlinx.coroutines.GlobalScope
-import kotlinx.coroutines.async
-import kotlinx.coroutines.delay
-import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.*
 import org.assertj.core.api.Assertions.assertThat
 import java.util.UUID
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.TimeUnit
 import kotlin.test.Test
 
+@OptIn(DelicateCoroutinesApi::class)
```

Comment:
which one is delicate?
please annotate only the affected method.

Reply:
All uses of `GlobalScope` are now considered 'delicate'. It's only a warning right now, but I'd imagine it will result in an error in future versions.
https://blog.jetbrains.com/kotlin/2021/05/kotlin-coroutines-1-5-0-released/#globalscope

That said, we should be able to remove `GlobalScope` because the tests are already using `runBlocking`.